embed.fnc

```diff
@@ -529,7 +529,9 @@
 :   's'  Static function, but function in source code has a Perl_ prefix:
 :
 :	 This is used for functions that have always had a Perl_ prefix, but
-:	 have been moved to a header file and declared static.
+:	 have been moved to a header file and declared static.  If those
+:	 functions are also inlined, the 'i' or 'I' flags should also be
```

Comment:
FWIW, this is not correct. the 's', 'S', 'i' and 'I' (capital I) flags are mutually exclusive. You can only have one. I think you have been mislead as there is a bug in embed.pl that uses the first one in the flags definition. I just happened to stumble on this bug myself independently of this ticket, and will push a PR to fix it in the next minutes. Note the code that looks like this: 

```
                $type = {
                    'S' => 'STATIC',
                    's' => 'STATIC',
                    'i' => 'PERL_STATIC_INLINE',
                    'I' => 'PERL_STATIC_FORCE_INLINE'
                }->{$1};
```

Reply:
See https://github.com/Perl/perl5/pull/20755